Demo doesn't work on Windows anymore

Probably the attach changes in https://github.com/xtermjs/xterm.js/pull/2422

```
buffer.js:480
      throw new ERR_INVALID_ARG_TYPE(
      ^

TypeError [ERR_INVALID_ARG_TYPE]: The "list[0]" argument must be one of type Array, Buffer, 
or Uint8Array. Received type string
    at Function.concat (buffer.js:480:13)     
    at Timeout.setTimeout [as _onTimeout] (D:\dev\Tyriar\xterm.js\demo\server.js:102:32)    
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)        
    at Timer.processTimers (timers.js:223:10) 
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! xterm@4.0.0 start: `node demo/start` 
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the xterm@4.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output 
above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Daniel\AppData\Roaming\npm-cache\_logs\2019-09-28T16_14_47_479Z-debug.log
```

--- a/demo/server.js
+++ b/demo/server.js
@@ -3,11 +3,8 @@ var expressWs = require('express-ws');
 var os = require('os');
 var pty = require('node-pty');
 
-/**
- * Whether to use binary transport.
- */
-const USE_BINARY = true;
-
+// Whether to use binary transport.
+const USE_BINARY = os.platform() !== "win32";
 
 function startServer() {
   var app = express();

